.github/workflows/codeql-analysis.yml

```
@@ -11,22 +13,28 @@ jobs:
   analyze:
     name: CodeQL Analysis
     runs-on: ubuntu-latest
+    
     steps:
+      - name: Setup Node.js 20
+        uses: actions/setup-node@v3
```

actionlint (1.7.4)

the runner of "actions/setup-node@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)